### ISSUE TITLE
neo-gui 2.6 stuck on block 2022689

### ISSUE THREAD

**Issue by chrisbednarski** [neo-project/neo-gui#215]
I've downloaded the neo-gui 2.6.0 (2.6.6596.21976)

I've left it on to download the blockchain, and it has been stuck on block 0/2022689/2022689  for two days now. I did not use the chain.acc file to speed the sych. Dowloaded all the blocks from the start.

Restarted the app about a dozen times now, Number of connected nodes changes between 0 to 10 depending on the time of the day.

If I open a wallet, the number goes up to 2022689/2022689/2022689

Let me know if you need any more information.

**Comment by chrisbednarski** [comment]
not sure if this is related, but I found the below in `error.log` file

```
System.ObjectDisposedException
Safe handle has been closed
   at System.Runtime.InteropServices.SafeHandle.DangerousAddRef(Boolean& success)
   at System.StubHelpers.StubHelpers.SafeHandleAddRef(SafeHandle pHandle, Boolean& success)
   at Microsoft.Win32.Win32Native.SetEvent(SafeWaitHandle handle)
   at System.Threading.EventWaitHandle.Set()
   at Neo.Network.LocalNode.Blockchain_PersistCompleted(Object sender, Block block)
   at System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   at Neo.Core.Blockchain.OnPersistCompleted(Block block)
   at Neo.Implementations.Blockchains.LevelDB.LevelDBBlockchain.PersistBlocks()
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.ThreadHelper.ThreadStart()
```

**Comment by chrisbednarski** [comment]
![2022689](https://user-images.githubusercontent.com/1804713/37563868-2d147358-2ade-11e8-8b16-197383a68d6b.png)

**Comment by jrsikors** [comment]
I think there are some nodes that aren't behaving properly... lots of people getting stuck on different block numbers.   Per #207 you can see my steps I've been taking , i'm testing around as well, though I'm not part of the NEO team.  I'm sure the team is aware of all the issues and hopefully working to resolve.

I might be stuck on block 2,037,647 myself during my test run.  I think instead of messing with the chain directory .. i might just try deleting the peers.dat when it gets stuck and try again .

**Comment by jrsikors** [comment]
so closing down neo, then deleting the peers.dat and opening neo-gui did make my stuck number start going up again ... though I'm pretty sure it will get stuck again.

Not a great workaround -- but better than nothing and/or deleting chain to start over

**Comment by jrsikors** [comment]
Also one other thing I thought about -- don't open your wallet file until you get your 2nd and 3rd numbers straightened out.  Opening your wallet just adds another unnecessary variable into the equation while you try to get the 2nd and 3rd numbers sync'd up.

**Comment by chrisbednarski** [comment]
I've downloaded `chain.acc` from https://github.com/neo-project/neo/issues/193 to bootstrap the download process. The file contains blocks up to 2022949. Trying that way

**Comment by Jdog88** [comment]
@chrisbednarski did that work?

**Comment by sandeepkrsony** [comment]
Same problem with me.. i sent 4.7 gas to poloniex when i track transaction.. there are showing UNSPENT..  it is almost 10 days completed..how to cofirm transaction.. what should i do..
block 2005191
txid- e1b75ec230c7f6ee87d60cf54d73a88952e9d3b11dd3b2b4761507ba49c2179a


  please please help anybody

**Comment by chrisbednarski** [comment]
still syncing 1883000/2043988

the last number keeps climbing ... so far so good, just need to wait for the block download to catch up to the current block height

**Comment by sandeepkrsony** [comment]
my block is 2005191. which is still unspent

**Comment by Jdog88** [comment]
@chrisbednarski yeah the block download is extra slow when it gets closer to the block height.  I find that if you see it stuck for a long time, close it and relaunch the wallet again, you will see the number jump and start running right away fast.

**Comment by sandeepkrsony** [comment]
how to close and relaunch again

**Comment by jrsikors** [comment]
just literally exit / close neo-gui.exe application ... then open neo-gui.exe application :smile:

**Comment by jrsikors** [comment]
I walked through the same process everyone else has to over the weekend... from scratch.

I deleted my chain folder
I deleted my peers.dat
I downloaded the latest chain.acc.zip and placed it in NEO-GUI folder ... didn't unzip it.

It took an hour to get to 0 / 1.8 mil / 1.8 mil
It then took another 23 hours to get the last 300,000 blocks
The only reason it took that long is because I forgot to close down NEO-GUI and re-open it before I went to bed... it was stuck when I went to bed.. so I missed out on a good 8-9 hours of sync time because of it

So if you are still having issues after days -- you aren't starting fresh and/or you aren't following instructions properly... or your computer is super slow (NEO-GUI.exe eats a lot of CPU during sync).

**Comment by chrisbednarski** [comment]
up to block 2045880/2045880 so all good now 

not sure where the bit of not unzipping `chain.acc.zip` came from
I unzipped the `chain.acc` into the same folder where `neo-gui.exe` is located and the last block number went up to 2022949 in a few minutes

**Comment by jrsikors** [comment]
it "came" from the instructions :smile: 
http://docs.neo.org/en-us/node/gui.html

and @erikzhang has said it as well

The 3rd number is just block height... the 2nd number is basically "where your computer is synced relative to block height".

So if you did the chain.acc.zip properly (and NOT unzipped it), your 2nd number would have gone up to about 1.8 million in an hour
